extensions/yabai/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "yabai",
+  "title": "Yabai",
+  "description": "Control Yabai using Raycast. Requires yabai to be installed.",
+  "icon": "yabai-icon-512px.png",
+  "author": "krzysztoff1",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "custom-yabai-command",
+      "title": "Run Custom Yabai Command",
+      "description": "Run custom Yabai command",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "name": "command",
+          "placeholder": "Command",
+          "type": "text",
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "toggle",
+      "title": "Toggle Yabai",
+      "description": "This command will start Yabai using `yabai --start-service` or stop it using `yabai --stop-service`.",
+      "mode": "no-view"
+    },
+    {
+      "name": "rotate",
+      "title": "Rotate",
+      "description": "Rotate the tree 90 degrees clockwise.",
+      "mode": "no-view"
+    },
+    {
+      "name": "mirror-x",
+      "title": "Mirror X",
+      "description": "Mirror the tree on the X axis.",
+      "mode": "no-view"
+    },
+    {
+      "name": "mirror-y",
+      "title": "Mirror Y",
+      "description": "Mirror the tree on the Y axis.",
+      "mode": "no-view"
+    },
+    {
+      "name": "balance",
+      "title": "Balance",
+      "description": "Balance the windows.",
+      "mode": "no-view"
+    },
+    {
+      "name": "create-space-and-focus",
+      "title": "Create & Focus New Space",
+      "subtitle": "yabai",
+      "description": "Creates a new workspace and focuses it.",
+      "mode": "no-view"
+    },
+    {
+      "name": "focus-space",
+      "title": "Focus Space",
+      "subtitle": "yabai",
+      "description": "Focuses the specified space.",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "type": "text",
+          "placeholder": "Space Number",
+          "required": true,
+          "name": "spaceIndex"
+        }
+      ]
+    },
+    {
+      "name": "destroy-space",
+      "title": "Destroy Current Space",
+      "subtitle": "yabai",
+      "description": "Destroys the currently focused space.",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "yabaiExecutablePath",
+      "description": "Path to the yabai executable.",
+      "type": "textfield",
+      "required": false,
+      "default": "./opt/homebrew/bin/yabai",
+      "title": "Path to the yabai executable",
+      "placeholder": "./opt/homebrew/bin/yabai",
+      "label": "By default, the extension will use the yabai executable from the homebrew installation. If you have yabai installed in a different location, you can specify it here."
+    }
```

Comment:
Should the `.` be there? Also, this doesn't work on Intel macs. I suggest doing something similar to:

https://github.com/raycast/extensions/blob/main/extensions/brew-services/package.json#L26
https://github.com/raycast/extensions/blob/main/extensions/brew-services/src/services.tsx#L28-L33

Reply:
Thanks! You are right, should be better now